noxfile.py

```diff
@@ -6,6 +6,19 @@
 import shutil
 import nox
 
+@nox.session(py="3")
+def translation(session):
+    session.install("-r", "requirements.txt")
+    target_dir = "locales"
+    session.run(
+        "sphinx-build", 
+        "-b",  # select a builder
+        "gettext",  # build gettext-style message catalogs (.pot file)
+        "source/",  # where the rst files are located
+        target_dir, # where to put the .pot file
+        "-d", 
+        ".nox/", # path to put the cache (.doctrees)
```

Comment:
*I think* this should be
```suggestion
        ".nox/.doctrees/", # path to put the cache
```